services/web/package.json

```diff
@@ -12,6 +12,10 @@
   "dependencies": {
     "@hookform/resolvers": "^3.9.1",
     "@radix-ui/react-accordion": "^1.2.0",
+    "@radix-ui/react-label": "^2.1.0",
```

Comment:
These correspond to the components we have downloaded from shadcn. But we are not using all of them. Investigate which ones are not used and follow shadcn documentation on how to remove them.